Ontologies.Mappings/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -139,17 +203,49 @@
       "OutputDtmi": "dtmi:com:willowinc:EnteringChilledWaterTemperatureSensor;1"
     },
     {
-      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Chilled_Water_Temperature_Setpoint;1",
+      "InputDtmi": "dtmi:mapped:core:Chilled_Water_Supply_Temperature_Setpoint;1",
       "OutputDtmi": "dtmi:com:willowinc:ChilledWaterTemperatureSetpoint;1"
     },
     {
-      "InputDtmi": "dtmi:org:w3id:rec:Company;1",
-      "OutputDtmi": "dtmi:com:willowinc:Company;1"
+      "InputDtmi": "dtmi:mapped:core:Chilled_Water_Usage_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:Sensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Class;1",
+      "OutputDtmi": "dtmi:com:willowinc:Class;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Close_Command;1",
+      "OutputDtmi": "dtmi:com:willowinc:ValveCloseActuator;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Close_Status;1",
+      "OutputDtmi": "dtmi:com:willowinc:PhaseBExportReactiveElectricalEnergySensor;1"
```

Comment:
Surely not! Maybe a CloseState?

<img width="425" alt="image" src="https://github.com/mapped/mapped-willow-mapping/assets/84034746/a1740d68-1c55-4267-8214-87af344e300e">

Reply:
Yeah, this was definitely a mistake. Thanks for catching this